ovos_utils/file_utils.py

```diff
@@ -98,13 +109,18 @@ def resolve_resource_file(res_name, root_path=None, config=None):
 
     Args:
         res_name (str): a resource path/name
+        root_path: Optional root path to check
         config (dict): mycroft.conf, to read data directory from
```

Comment:
For a future release, consider making this a required arg and dropping ovos-config fallback

Reply:
#130